tests/phpunit/Helpers/WC_Subscriptions.php

```diff
@@ -25,11 +28,15 @@ class WC_Subscriptions {
 	/**
 	 * wcs_get_subscription mock.
 	 *
-	 * @var function
+	 * @var callable
 	 */
 	public static $wcs_get_subscription = null;
 
-	public static function set_wcs_get_subscription( $function ) {
-		self::$wcs_get_subscription = $function;
+	/**
+	 * @param callable $calback Function to call when wcs_get_subscription is called.
+	 * @return void
+	 */
+	public static function set_wcs_get_subscription( $calback ) {
+		self::$wcs_get_subscription = $calback;
```

Comment:
Nit: there is a typo in `$calback`.
```suggestion
	 * @param callable $callback Function to call when wcs_get_subscription is called.
	 * @return void
	 */
	public static function set_wcs_get_subscription( $callback ) {
		self::$wcs_get_subscription = $callback;
```